be/src/exec/dict_decode_node.cpp

```diff
@@ -99,6 +102,13 @@ Status DictDecodeNode::open(RuntimeState* state) {
         auto dict_iter = global_dict.find(need_encode_cid);
         auto dict_not_contains_cid = dict_iter == global_dict.end();
 
+        if (dict_not_contains_cid) {
```

Comment:
why we add these code

Reply:
global dict expression is lazy compute, will compute when frist used